### PR TITLE
HW1 - al4007

### PR DESCRIPTION
**Pull request by luo-anthony**

### CODE REVIEW

File: al4007_projects/AudioMoth.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# AudioMoth
```
Comment (timkpaine) [inline al4007_projects/AudioMoth.md:1]: move to `projects/cpp/AudioMoth.md`

File: al4007_projects/hybriddatastructures.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# hybrid-data-structures
```
Comment (timkpaine) [inline al4007_projects/hybriddatastructures.md:1]: move to `projects/cpp/hybriddatastructures.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: al4007_projects/hybriddatastructures.md (hunk 2 above)
Comment (luo-anthony) [reply al4007_projects/hybriddatastructures.md]: fixed in next commit

File: al4007_projects/AudioMoth.md (hunk 1 above)
Comment (luo-anthony) [reply al4007_projects/AudioMoth.md]: fixed in next commit

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**